Game.html

```diff
@@ -1,15 +1,44 @@
-<script>
-var guess,
+<script type="text/javascript">
+/* This is a Global variable that will control the looping of 
+the game. */
+var yourLatestGuess,
+		retry = true;
 
-guess = prompt ("How many holes in a bowling ball");
+/* This function was created to list the tasks needed to perform
+the guessing game. There are three scenarios that can occur in the
+game depending on the user's input. */
+var theGame = function() {
+	//3 variables assigned that are local to this function
+	var userGuess,
+		answer = 3,
+		message;
 
-if(guess==3)
-{
-	alert("Correct");
-}
-else
+	//This checks to see if the user's input equals the answer
+	if (yourLatestGuess == answer) {
+		retry = false;
+		message = alert("Correct.");
+	}
+	//This checks whether the user wants to quit the game
+	else if (yourLatestGuess == 'q') {
+		retry = false;
+		message = alert("Sorry to see you go.");
+	}
+	//This displays if user input is incorrect but want to try again
+	else {
+		message = prompt("Wrong. Keep guessing until you get the answer or enter 'q' to quit.");
+	}
+
+	return message;
+
+};
 
-{
-	alert("Wrong");
+//This is the initial prompt the user sees upon start of game
+yourLatestGuess = prompt("Can you guess how many holes are in a bowling ball?");
+
+/* The function theGame will keep getting prompted to the user as long as he/she continues to not get the answer
+and does not input 'q' to quit. */
+while(retry == true) {
+	yourLatestGuess = prompt(theGame());
```

Comment:
since you prompt/alert inside of theGame, do you need to prompt here as well?
could you just do this:
```
while(retry == true) {
  theGame();
}
```
also, no need to compare to true, you can do this:
`while(retry)` instead of `while(retry == true)`